### PR TITLE
Bug 1829363- Update Home.md

### PR DESCRIPTION
**Pull request by GammaBurst101**
The links in docs/Home.md of firefox-focus used to point at old links.
Shared docs link pointed to the archived repo. Updated it to link to current repo
Releases link pointed to archived focus repo. It now points to current repo's releases
Same is done for others except for good first issues. For it I just linked to bugzilla

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1829363

### CODE REVIEW

File: focus-android/docs/Home.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ https://www.mozilla.org/firefox/focus/
...
 * Google Play - Firefox Klar: https://play.google.com/store/apps/details?id=org.mozilla.klar
-* APKs: https://github.com/mozilla-mobile/focus-android/releases
+* APKs: https://github.com/mozilla-mobile/firefox-android/releases
 
 ## Contribute
 
-* [Contributing (Writing code, translating the app, testing the app)](https://github.com/mozilla-mobile/shared-docs/blob/master/android/CONTRIBUTING.md)
-* [List of good first issues for new contributors](https://github.com/mozilla-mobile/focus-android/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+* [Contributing (Writing code, translating the app, testing the app)](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/CONTRIBUTING.md)
+* [List of good first issues for new contributors](https://bugzilla.mozilla.org/describecomponents.cgi?product=Focus)
```
Comment (MatthewTighe) [inline focus-android/docs/Home.md:23]: I am happy with where the link points, but could we update the link text to be "List of issues"

File: focus-android/docs/Home.md [hunk 2]
```diff
@@ -29,13 +29,13 @@ https://www.mozilla.org/firefox/focus/
...
-* [Automation](https://github.com/mozilla-mobile/shared-docs/blob/master/android/automation.md)
-* [Content blocking](https://github.com/mozilla-mobile/focus-android/wiki/Content-blocking)
-* [Multisession architecture](https://github.com/mozilla-mobile/focus-android/wiki/Multisession-architecture)
+* [Architecture decisions](https://github.com/mozilla-mobile/firefox-android/blob/main/focus-android/docs/Architecture-Decisions.md)
+* [Automation](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/automation.md)
+* [Content blocking](https://github.com/mozilla-mobile/firefox-android/blob/main/focus-android/docs/Content-blocking.md)
+* [Multisession architecture](https://github.com/mozilla-mobile/firefox-android/blob/main/focus-android/docs/Multisession-architecture.md)
 * [Localization](https://github.com/mozilla-mobile/focus-android/wiki/Localization)
-* [Telemetry](https://github.com/mozilla-mobile/focus-android/wiki/Telemetry)
+* [Telemetry](https://github.com/mozilla-mobile/firefox-android/blob/d8939c35d509cb4839030a318db97025674aea20/focus-android/docs/Telemetry.md)
```
Comment (MatthewTighe) [inline focus-android/docs/Home.md:37]: Can we use `blob/main` for this link as well?

File: focus-android/docs/Home.md [hunk 3]
```diff
@@ -29,13 +29,13 @@ https://www.mozilla.org/firefox/focus/
...
 
 ## Product Management
-* [Roadmap](https://github.com/mozilla-mobile/focus-android/projects/2)
+* [Roadmap](https://github.com/mozilla-mobile/firefox-android/projects?query=is%3Aopen)
```
Comment (MatthewTighe) [inline focus-android/docs/Home.md:40]: We can just remove this bullet entirely since we are not doing explicit project tracking in Github or Bugzilla.

**Review by MatthewTighe — CHANGES_REQUESTED** [review]
Hi there, thanks a ton for your contribution! Had just a few small things to work out but this otherwise LGTM

File: focus-android/docs/Home.md (hunk 1 above)
Comment (GammaBurst101) [reply focus-android/docs/Home.md]: Sure

File: focus-android/docs/Home.md (hunk 2 above)
Comment (GammaBurst101) [reply focus-android/docs/Home.md]: Changed it to use blob/main in latest commits

**Review by MatthewTighe — CHANGES_REQUESTED** [review]
These updates are looking ready to land, but will need a couple small things first. Thanks for your patience in getting this together!

1. Please squash your commits so that there's only 1
2. The commit messages and names might require a space between the bug number and the following dash though I am not 100% sure of that. Something like: `Bug 1829363 - Update Home.md`

**Review by MatthewTighe — APPROVED** [review]
LGTM! Thanks again for your contribution 🎉

**Comment by MatthewTighe** [comment]
bors try

**Comment by MatthewTighe** [comment]
bors retry

**Comment by MatthewTighe** [comment]
The CI failures should be unrelated to your changes. Could you try rebasing onto our main branch and pushing again? Thanks

**Comment by MatthewTighe** [comment]
bors retry

**Comment by MatthewTighe** [comment]
Sorry this is taking so long. Looks like our CI pipeline is a little flaky recently 😅 I will just go ahead and land this patch if it continues to fail since your changes are clearly unrelated to the failures

**Comment by MatthewTighe** [comment]
It looks like we'll need to manually rebase this again. I'd be happy to start doing that for you if you'd like, but it will require force-pushing to your fork's main branch. I don't think you'd need to change anything for me to be able to do that, but wanted to give you a heads up.

Otherwise, I'm happy to let you continue rebasing until this problem is resolved

**Comment by GammaBurst101** [comment]
Thank you for offering to rebase it for me. Please go ahead and handle the manual rebase.

**Comment by MatthewTighe** [comment]
bors retry

**Pull request merged**